img-src/Dockerfile

```diff
@@ -1,16 +1,10 @@
-FROM debian:buster
+FROM alpine
 
-RUN apt update
-RUN apt upgrade -y
-RUN DEBIAN_FRONTEND=noninteractive apt install -y \
-    bash \
+RUN apk add -U \
     iptables
-RUN apt clean
```

Comment:
You should add the equivalent for `apk` to have the image smaller.
According to [this thread](https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk) the best way is to run `apk add` with the `--no-cache` parameter